spec/models/google_analytics/data_tracking_spec.rb

```diff
@@ -13,10 +13,23 @@ module GoogleAnalytics
           allow(described_class).to receive(:adapter).and_return('Adapter')
         end
 
-        %w(staging production).each do |host|
-          it "returns true when host is #{host}" do
-            allow(RailsHost).to receive(:env).and_return(host)
-            expect(described_class.enabled?).to be_truthy
+        context 'with usage tracking accepted' do
+          %w(staging production).each do |host|
+            it "returns true when host is #{host}" do
+              allow(RailsHost).to receive(:env).and_return(host)
+              allow(described_class).to receive(:usage_name).and_return('true')
+              expect(described_class.enabled?).to be_truthy
+            end
+          end
+        end
+
+        context 'with usage tracking rejected' do
+          %w(staging production).each do |host|
```

Comment:
we were looking to unfurl these rspec loops. can use context blocks with before hooks to set these
```
context 'when on staging' do
  before { allow(RailsHost).to receive(:env).and_return('staging') }
  ...
end
```